presto-manager-controller/src/main/java/com/teradata/prestomanager/controller/ApiResponseGenerator.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.teradata.prestomanager.controller;
+
+import com.google.gson.Gson;
+import com.teradata.prestomanager.common.ApiRequester;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import static com.teradata.prestomanager.common.ExtendedStatus.MULTI_STATUS;
+import static javax.ws.rs.core.Response.Status.BAD_REQUEST;
+import static javax.ws.rs.core.Response.Status.INTERNAL_SERVER_ERROR;
+
+public final class ApiResponseGenerator
```

Comment:
I would call this something more indicative of its role as a call sender; it doesn't generate responses, it sends responses to some group of nodes.

Maybe `ApiCallForwarder` (though it isn't limited to forwarding) or `RequestMultiplexer` (or something less silly-sounding).